src/handleScroll.ts

```diff
@@ -90,6 +90,9 @@ export const handleScroll = (
   let availableScrollTop = 0;
 
   do {
+    if (target instanceof ShadowRoot) {
```

Comment:
"when" this happens?
Description of commit is about "mobile", but how it's related?

Reply:
If the event comes from within a ShadowRoot, then the target's .parentNode() up the tree will eventually be the ShadowRoot, and we need to step out of it with .host (because a ShadowRoot's parentNode is null). I'm not exactly sure why this fixes it, but touch scrolling doesn't work without this in my use case. I already wanted to do this regardless, because it seems like the "right" thing to do with regards to ShadowDOM, but it also ended up fixing the issue on mobile.